src/portable/renesas/usba/dcd_usba.c

```diff
@@ -292,6 +310,13 @@ static void process_setup_packet(uint8_t rhport)
   setup_packet[1] = USB0.USBVAL;
   setup_packet[2] = USB0.USBINDX;
   setup_packet[3] = USB0.USBLENG;
+#if TU_BYTE_ORDER==TU_BIG_ENDIAN
+ #if defined(__CCRX__)
```

Comment:
there is no need to check for compiler, if there is other one, it will be handled by the tu_le16toh().

Reply:
Yes, I agree with you